### PR TITLE
Streamline Messages bundle classes

### PR DESCRIPTION
**Pull request by chanseokoh**
Making the classes consistent.

### CODE REVIEW

**Comment by chanseokoh** [comment]
Okay, this reveals a test failure. I'll fix it.

**Comment by chanseokoh** [comment]
Wow, [`MessageFormat.format()` removes single quotes](http://stackoverflow.com/questions/189889/java-messageformat-how-can-i-insert-values-between-single-quotes). Another reason to be consistent; we should always call `MessageFormat.format()`.

File: plugins/com.google.cloud.tools.eclipse.usagetracker/src/com/google/cloud/tools/eclipse/usagetracker/messages.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
 OPT_IN_DIALOG_TITLE=Google Cloud Tools for Eclipse
 OPT_IN_DIALOG_TEXT=Share anonymous usage statistics of Cloud Tools for Eclipse with Google?
 OPT_IN_BUTTON=Share
-OPT_OUT_BUTTON=Don't Share
\ No newline at end of file
+OPT_OUT_BUTTON=Don''t Share
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.usagetracker/src/com/google/cloud/tools/eclipse/usagetracker/messages.properties]: I wonder if this should be _Do Not Share_?

Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.usagetracker/src/com/google/cloud/tools/eclipse/usagetracker/messages.properties]: That sounds more formal. +1

**Pull request merged**